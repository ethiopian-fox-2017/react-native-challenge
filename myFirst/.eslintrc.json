{
  "parser": "babel-eslint",
  "env": {
      "browser": true
  },
  "plugins": [
      "react"
  ],
  "extends": [
      "eslint:recommended",
      "plugin:react/recommended"
  ],
  "rules": {
      // overrides
  },
  "globals": {
    "test": true,
    "it": true,
    "describe": true,
    "expect": true,
    "afterEach": true
  }
}
