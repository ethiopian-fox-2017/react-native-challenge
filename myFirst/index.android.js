/**
 * Sample React Native App
 * https://github.com/facebook/react-native
 * @flow
 */

import {
  AppRegistry,
} from 'react-native';
import { StackNavigator } from 'react-navigation';

import Home from './src/components/Home';
import About from './src/components/About';

const myFirst = StackNavigator({
    Home: {screen: Home},
    About: {screen: About},
  },{
    headerMode: 'none',
  }

);




AppRegistry.registerComponent('myFirst', () => myFirst);
