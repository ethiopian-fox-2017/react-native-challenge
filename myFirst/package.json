{
  "name": "myFirst",
  "version": "0.0.1",
  "private": true,
  "scripts": {
    "start": "node node_modules/react-native/local-cli/cli.js start",
    "test": "jest"
  },
  "dependencies": {
    "axios": "^0.16.1",
    "babel-eslint": "^7.2.3",
    "enzyme": "^2.8.2",
    "eslint-plugin-import": "^2.2.0",
    "eslint-plugin-react": "^6.10.3",
    "fetch-mock": "^5.10.0",
    "native-base": "^2.1.3",
    "react": "16.0.0-alpha.6",
    "react-addons-test-utils": "^15.5.1",
    "react-dom": "^15.5.4",
    "react-native": "0.44.0",
    "react-navigation": "^1.0.0-beta.9",
    "react-redux": "^5.0.4",
    "redux": "^3.6.0",
    "redux-logger": "^3.0.1",
    "redux-mock-store": "^1.2.3",
    "redux-thunk": "^2.2.0",
    "sinon": "^2.2.0"
  },
  "devDependencies": {
    "babel-jest": "19.0.0",
    "babel-preset-react-native": "1.9.1",
    "eslint": "^3.19.0",
    "eslint-config-standard": "^10.2.1",
    "eslint-plugin-promise": "^3.5.0",
    "eslint-plugin-standard": "^3.0.1",
    "jest": "19.0.2",
    "react-test-renderer": "16.0.0-alpha.6"
  },
  "jest": {
    "preset": "react-native",
    "setupFiles": [
      "./jest/setup.js"
    ],
    "transformIgnorePatterns": [
      "/node_modules/(?!native-base)/"
    ]
  }
}
