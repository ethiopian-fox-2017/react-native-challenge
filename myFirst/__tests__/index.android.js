import 'react-native';
import React from 'react';
import myFirst from '../index.android.js';
import { shallow } from 'enzyme';
// Note: test renderer must be required after react-native.


test('renders correctly', () => {
  const tree = shallow(
    <myFirst />
  );

  expect(tree).toMatchSnapshot();
});
