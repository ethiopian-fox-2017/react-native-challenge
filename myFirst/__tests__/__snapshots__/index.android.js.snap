// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <myFirst />,
  "nodes": Array [
    <myFirst />,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": null,
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <myFirst />,
}
`;
